etc/ATSCam/primary_hdu.header

```diff
@@ -27,13 +33,16 @@ OBSERVER=                      / Observer name(s)
 PROPOSER=                      / Proposal Principle Investigator
 PROPID  =                      / Proposal ID
 SEQFILE =                      / Name of Sequencer File
```

Comment:
Do we know what this really means?  @r-owen might know what we would like in here.

Reply:
This is sequencer file used by camera as part of its configuration. Previous chatter on #lsst-ats suggests that it will be need/wanted.